packages/examples/examples/typescript/src/index.ts

```diff
@@ -0,0 +1,18 @@
+wallet.registerRpcMessageHandler(async (originString, requestObject) => {
+  switch (requestObject.method) {
+    case 'hello':
+      return wallet.request({
+        method: 'snap_notify',
+        params: [
+          {
+            type: 'native',
+            message: `Hello, ${originString}!`,
+          },
+        ],
+      });
+    default:
+      throw new Error('Method not found.');
+  }
+});
+
+export {};
```

Comment:
Why is this needed?

Reply:
Otherwise ESLint will complain about it being a script or something

Reply:
Could we just disable that? 🤔

Reply:
The import plugin authors [recommend](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/unambiguous.md#rule-details) dealing with it in this way. We could fiddle around with our config or disable the rule at the top of this file, but we could equally well just do this:

```suggestion
// Just for compatibility with the ESLint import/unambiguous rule.
export {};
```